share/docker/Dockerfile

```diff
@@ -78,13 +78,6 @@ RUN apt-get -qq update && \
   apt-get -qq clean && \
   rm -rf /var/lib/apt/lists/*
 
-# Install Forrest in /usr/local/apache-forrest
-RUN curl -L -s "http://www.apache.org/dyn/closer.lua?action=download&filename=/forrest/apache-forrest-0.9-sources.tar.gz" | tar -xzf - -C /usr/local/ && \
-    curl -L -s "http://www.apache.org/dyn/closer.lua?action=download&filename=/forrest/apache-forrest-0.9-dependencies.tar.gz" | tar -xzf - -C /usr/local/ && \
-    mv /usr/local/apache-forrest-0.9 /usr/local/apache-forrest && \
-    cd /usr/local/apache-forrest/main && \
-    ./build.sh
-
 # The solution for https://issues.apache.org/jira/browse/PIG-3906
```

Comment:
Few things:
Forrest was included in the docker file to generate the documentation. Don't we need that?
If it is incompatible with JDK 11 then why not have a condition around it to only run under JDK 8?
In case we choose to remove it completely then the next lines should be removed too.

Reply:
Good point @nielsbasjes 

So we don't generate the docs in the CI. This is only done when we do the actual release. I think it would make sense to revamp the docs, but this will be a lot of work.

Currently, the generation of docs is quite straightforward. For the 1.9.0 release, I did this outside of Docker. The only dependencies are ANT and Forrest: https://cwiki.apache.org/confluence/display/AVRO/How+To+Release

Since we're not running any checks against the docs, I think it would make sense to omit this from the image as well. WDYT?

Reply:
I agree we do not need it in CI.
The reason I put in in the Docker image is that getting forrest to run correctly is a tricky thing.
The docker image was created so that devs can run the build in a preconfigured build environment locally with minimal hassle.
I propose: We have a base image that is suitable for CI and an additional layer docker image that includes this forrest fragment that can be used by devs who simply do "./build.sh docker" 
For now: Remove it (so we have a working CI build again) but leave it commented in so a we still have close by how to get it up and running.

Reply:
Just leave it as you have now. I'll look into making a working dev environment again.

Reply:
https://issues.apache.org/jira/browse/AVRO-2408

Reply:
Thanks!